docs/cristin-proxy-swagger.yaml

```diff
@@ -46,6 +46,14 @@ paths:
             type: string
             pattern: '.+'
           example: 'Helse'
+        - name: organization
+          in: query
+          description: Search projects by organization id.
+          required: false
+          schema:
+            type: string
+            pattern: '.+'
+          example: 'https://api.dev.nva.aws.unit.no/cristin/organization/20202.0.0.0'
```

Comment:
Are you using an uri as a query parameter? Is it not better to only use the last part 20202.0.0.0? I dont think it is wise to use 2 uris in the same request

Reply:
It's the URI which is the full and unambiguous id which precisely and persistent identifies the Organization - and we don't have to operate on partial identifers. 
Maybe how and what to use for id should be discussed for the whole project?